pkg/cloud/services/elb/loadbalancer.go

```diff
@@ -255,7 +255,7 @@ func (s *Service) getAPIServerClassicELBSpec() (*infrav1.ClassicELB, error) {
 			},
 		},
 		HealthCheck: &infrav1.ClassicELBHealthCheck{
-			Target:             fmt.Sprintf("%v:%d", infrav1.ClassicELBProtocolSSL, 6443),
```

Comment:
I’d rather update the listeners/health checks to use custom order/set of SSL ciphers

Reply:
As far as ELB goes, listeners != health checks in this regards. It is TCP passthrough for the listener. The healthcheck spec just ensures we do a TLS handshake.

Reply:
Verified that AWS only supports non-elliptic curve handshake tests for ELBv1.

Reply:
sgtm